arch/rv32i/src/syscall.rs

```diff
@@ -144,7 +144,7 @@ impl kernel::syscall::UserspaceKernelBoundary for SysCall {
           //  0*4(sp): app s0   <- new stack pointer
           // ```
 
-          addi sp, sp, -33*4  // Move the stack pointer down to make room.
+          addi sp, sp, -34*4  // Move the stack pointer down to make room.
```

Comment:
Need to update the comment above this since the format has changed.

Reply:
Updated.